_posts/2016-09-26-Debezium-0-3-2-Released.adoc

```diff
@@ -0,0 +1,15 @@
+---
+layout: post
+title: Debezium 0.3.2 Released
+date:  2016-09-26 10:19:59 -0600
+tags: [ releases, mysql, mongodb, docker ]
+author: rhauch
+---
+
+We're happy to announce that **Debezium 0.3.2** is now available for use with Kafka Connect 0.10.0.1. This release contains a handful of bug fixes and minor improvements for the link:/docs/connectors/mysql/[MySQL connector] and link:/docs/connectors/mongodb/[MongoDB connector]. The MySQL connector better handles `BIT(n)` values and http://dev.mysql.com/doc/refman/5.7/en/date-and-time-types.html[zero-value] date and timestamp values. This release also improves the log messages output by the MySQL and MongoDB connectors to better represent the ongoing activity when consuming the changes from the source database. See the link:/docs/releases/[release notes] for specifics.
+
+We've also updated the https://hub.docker.com/r/debezium/[Debezium Docker images] labelled `0.3` and `latest`, which we use in our link:/docs/tutorial/[tutorial]. We've also updated the tutorial to use the https://docs.docker.com/engine/installation/[latest Docker installations] on Linux, Windows, and OS X.
+
+Thanks to Akshath, Colum, Emmanuel, Konstantin, Randy, RenZhu, Umang, and others for their help with the release, issues, discussions, contributions, and questions!
+
++++<!-- more -->+++
```

Comment:
This marker at EOF makes no sense? (same for next file and some others).

Reply:
If this is not included, the excerpt will still be shown but the formatting will be stripped.  Could look for alternate approach at some point